gojq.hcl

```diff
@@ -0,0 +1,19 @@
+description = "gojq is a pure Go implemention of jq. jq is like sed for JSON data - you can use it to slice and filter and map and transform structured data with the same ease that sed, awk, grep and friends let you play with text."
+binaries = ["gojq"]
+test = "gojq --version"
+
+linux {
+  source = "https://github.com/itchyny/gojq/releases/download/v${version}/gojq_v${version}_linux_${arch}.tar.gz"
```

Comment:
This and below can be consolidated by replacing `linux`/`darwin` with `${os}`

Reply:
One is `.tar.gz` the other is `.zip`

Reply:
Brilliant! 🤦

Reply:
i can still replace the os with ${os} but I did it this way because that part of the name was constant for linux & darwin, and they still need to be separate for the extension.

Reply:
Yep makes total sense and agreed.